lesson_06/script.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+const ul = document.getElementById("js-list");
+const fragment = document.createDocumentFragment();
+const data = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const promise = new Promise((resolve) => {
+  setTimeout(() => {
+    resolve(data);
+  }, 3000);
+});
+promise.then((value) => {
+  for (let i = 0; i < value.length; i++) {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const img = document.createElement("img");
+    a.href = value[i].to;
+    a.text = value[i].text;
```

Comment:
Question
I did not know you could write this way. 
I would like to know if you have any references. 
I used to write "a.textContent".

Reply:
My apologies. I did not know there was a property called .textContent. I will study it.